Sources/NIO/BaseSocket.swift

```diff
@@ -18,6 +18,11 @@ import NIOConcurrencyHelpers
 import let WinSDK.INVALID_SOCKET
 #endif
 
+public enum SocketSetupError: Error {
```

Comment:
This should be a `struct`, not an `enum`.

Reply:
fixed